config/rbac/role.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   creationTimestamp: null
-  name: clusterlogging-operator
+  name: cluster-logging
```

Comment:
I think this is better but does it break upgrade?

Reply:
I'm reverting, all the clusterlogging => cluster-logging changes. It is more consistent but best not mix it into this commit.